### ISSUE TITLE
Bitrate for WLAN is not shown

### ISSUE THREAD

**Issue by Garionion** [greshake/i3status-rust#408]
So as the title says, the Bitrate of the connected WLAN is not displayed. the config section looks like this:
```[[block]]
block = "net"
device = "wlp4s0"
ssid = true
bitrate = true
```

Am I doing something wrong?

**Comment by nicholasfagan** [comment]
Can you show the output of the command:
    `iw dev wlp4s0 link | awk '/tx bitrate/ {print $3" "$4}'`
This is the command used to grab the bit rate.

**Comment by Garionion** [comment]
So, I actually hadn't installed `iw`, but it would be nice, if this dependencie would standing at the block documentation.

Is it normal., that some of the time, it displays only 6.0 Mbit whereas I have constantly a bitrate of 866.7Mbit (which it displays most of the time)

**Comment by nicholasfagan** [comment]
The bitrate is a measure of the current max bandwidth, as configured by the network drivers, based on the strength of the signal/amount of noise in the connection (It's a measure of how strong the connection is). It will only change when your connection changes, for example if you moved further from your router.
This is not to be confused with the current upload/download speed, which is measuring how much data is actually flowing.

In short, yes it is normal for your bitrate to stay the same most of the time.

Also, I'm curious to know what distro are you using? I though most (if not all) systems have `iw` as part of the kernel.

**Comment by Garionion** [comment]
> This is not to be confused with the current upload/download speed, which is measuring how much data is actually flowing.
Don't worry, I didn't ^^

> Also, I'm curious to know what distro are you using?
I am using a freshly installed Arch

**Comment by nicholasfagan** [comment]
I always thought `iw` was in the `base` or `base-devel` package group, but I just checked and I was mistaken. I'll see if there are any other dependencies like this that are not mentioned in the documentation, and make an issue for it.

**Comment by atheriel** [comment]
Yep, changing the docs is the right thing to do -- you can open a PR instead of an issue if you like. We do miss things.

Closing this, unless I've misunderstood that it is resolved.
